extension/src/plots/index.ts

```diff
@@ -172,8 +178,32 @@ export class Plots extends BaseRepository<TPlotsData> {
     return this.sendPlots()
   }
 
-  private setHasCustomSelection() {
-    this.paths.setHasCustomSelection(this.paths.getTerminalNodes().length > 20)
+  private setHasCustomSelection(types: Set<PathType> | undefined) {
+    if (!types || types.has(PathType.COMPARISON)) {
+      const comparisonNodes = this.paths.getTerminalNodesByType(
+        PathType.COMPARISON
+      )
+      this.paths.setHasCustomSelection(
+        comparisonNodes.length > 20,
```

Comment:
[Q] Could we have this as a boolean value that is only set to true once? As a path is toggled (on or off) we could set the boolean to false. Would that work/help to simplify the code? Something like `hasUserSelectedPlots`?

Reply:
It might actually make everything easier to follow/simplify the logic if we invert the flag. 
Change the flag to `showingDefaultNumberPlots` and only set it to `true` when previously there were < 20 plots available in a section and now there are > 20 plots. Any subsequent change in selection could then set the flag to `false`.

☝🏻 might be easier to chat about this on a call.

Reply:
This definitely needs to be simplified because I try to explain my reasoning and I keep backtracking.